search/signals.py

```diff
@@ -12,26 +12,32 @@
     Employment,
     Profile,
 )
+from dashboard.models import (
+    Certificate,
+    Enrollment,
+)
 from search.tasks import index_users, remove_user
 
 log = logging.getLogger(__name__)
 
 
 @receiver(post_save)
-def handle_profile_update(sender, **kwargs):  # pylint: disable=unused-argument
-    """Update index when a Profile, Education, or Employment is updated."""
+def handle_update(sender, **kwargs):  # pylint: disable=unused-argument
+    """Update index when a relevant model is updated."""
     instance = kwargs["instance"]
```

Comment:
if you define the signal handler with an arg called `instance`, it will be unpacked from `kwargs` automatically